.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,15 +8,17 @@
 
 [Link to xdb-apis/xdb-golang-sdk PRs if it's on top of any API changes]
 
-- API change: ...
-- Golang SDK change: ...
+- API change link: ...
+- Golang SDK change link: ...
 - Server Component 1: ...
 - Server Component 2: ...
 
 ## How to test this pull request?
 
 [If writing Integration test in Golang SDK repo, please provide link to the pull request of Golang SDK Repo]
 
+[It's recommended to write integration test in Golang SDK repo, and enabled in this server repo first, 
```

Comment:
@duoertai @zklgame WDYT we use this approach to add integ tests? :
1. a PR to add test in go-sdk, but not enabled in go-sdk
2. only enabled in server PR and make sure local passing
3. merged the go-sdk PR(new tests without enabling)
4. merged the server PR
5. finally create a new sdk PR to enable the test in SDK(now it will pass because the server is already merged and publish the new image)
6. A

Reply:
Java SDK is my preference. Can I write the integ in Java SDK first to test the server implementation?

Reply:
Good point. I added another instructions for other SDK for options.